ntfy/backends/pushover.py

```diff
@@ -17,6 +25,25 @@ def notify(title, message, user_key,
     if device:
         data['device'] = device
 
+    if sound:
+        data['sound'] = sound
+
+    if priority != 0 and priority <= 2 and priority >= -2:
+        data['priority'] = priority
+
+        # Expire, Retry, and Callback only apply to an Emergency Priority
+        if priority == 2:
```

Comment:
Raise errors or warn about Expire, retry & callback being ignored in an else clause if they're set.